### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by JonathanLeeWH**
Update README.adoc by changing Address Book (Level 4) to MediTabs
Add description of MediTabs
Add benefits of MediTabs
Remove Learning Outcomes link
Add acknowledgement to the original source code AddressBook-Level4 project by SE-EDU initiative
Add Netlify integration to preview documentation

### CODE REVIEW

**Comment by JonathanLeeWH** [comment]
Fixed some formatting with regards to a section title and also I apologise I realised I spelt "amendments wrongly in my commit message"

File: README.adoc [hunk 1]
```diff
@@ -13,24 +13,27 @@ ifndef::env-github[]
...
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+== Introduction to MediTabs
```
Comment (jtankw3) [inline README.adoc]: Might want to make the marketing blurb more convincing by adding some of the key features such as the warning and sorting features.

File: README.adoc [hunk 2]
```diff
@@ -13,24 +13,27 @@ ifndef::env-github[]
...
 * <<DeveloperGuide#, Developer Guide>>
-* <<LearningOutcomes#, Learning Outcomes>>
 * <<AboutUs#, About Us>>
 * <<ContactUs#, Contact Us>>
 
 == Acknowledgements
 
+* Original source code: https://github.com/nus-cs2103-AY1819S2/addressbook-level4[AddressBook-Level4 project] by https://github.com/se-edu/[SE-EDU initiative]
+
 * Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
```
Comment (jtankw3) [inline README.adoc]: Maybe remove the word 'sample' since we are a 'real' product

Comment (JonathanLeeWH) [reply README.adoc]: @jtankw3 Resolved in the next commit. Thank you for spotting it.

File: README.adoc (hunk 1 above)
Comment (JonathanLeeWH) [reply README.adoc]: @jtankw3 This is indeed a good suggestion. I have added some key features section which does not list all the key features but only some so as to give users a sneak peek at what our application offers.

**Pull request merged**